src/SFA.DAS.EAS.Web.IntegrationTests/LegalEntitiesControllerTests/WhenIGetLegalEntitiesWithUnknownIds.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SFA.DAS.EAS.Account.Api.Controllers;
+using SFA.DAS.EAS.Account.Api.Types;
+using SFA.DAS.EAS.Account.API.IntegrationTests.TestUtils.ApiTester;
+
+namespace SFA.DAS.EAS.Account.API.IntegrationTests.LegalEntitiesControllerTests
+{
+    [TestFixture]
+    public class WhenIGetLegalEntitiesWithUnknownIds
+    {
+        private ApiIntegrationTester _tester;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _tester = new ApiIntegrationTester();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            _tester.Dispose();
+        }
+
+        [Test]
+        public async Task ThenTheStatusShouldBeNotFound_ByHashedId()
+        {
+            // Arrange
+            var callRequirements = new CallRequirements($"api/accounts/MADE*UP*ID/legalentities");
+
+            // Act
+            await _tester.InvokeGetAsync<ResourceList>(callRequirements);
+
+            // Assert
+            Assert.Pass("Verified we got http status NotFound");
```

Comment:
We don't actually check for not found so I don't think we can assert a Pass here - previously we had a verification set up when creating the callRequirements object. In other tests we are doing something like account.ExpectStatusCodes(HttpStatusCode.OK);

Reply:
They're extension methods on the return type from InvokeGetAsync<>() so if you assign the result to a local variable then you should see the available methods in intellisense.

Reply:
Changes made as requested.